atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -350,15 +436,17 @@ public void hostsAreNotRemovedOrAddedWhenRefreshIsDisabled() {
 
         setCassandraServersTo(CASS_SERVER_1);
         createClientPool();
-        assertThat(poolServers).containsExactlyInAnyOrder(CASS_SERVER_1, CASS_SERVER_2);
+        assertThat(poolServers.keySet()).containsExactlyInAnyOrder(CASS_SERVER_1, CASS_SERVER_2);
 
         setCassandraServersTo(CASS_SERVER_1, CASS_SERVER_2, CASS_SERVER_3);
         deterministicExecutor.tick(POOL_REFRESH_INTERVAL_SECONDS, TimeUnit.SECONDS);
-        assertThat(poolServers).containsExactlyInAnyOrder(CASS_SERVER_1, CASS_SERVER_2);
+        assertThat(poolServers.keySet()).containsExactlyInAnyOrder(CASS_SERVER_1, CASS_SERVER_2);
     }
 
     @Test
     public void shutsDownHostsBeyondAbsenceTolerance() {
+        when(cassandraTopologyValidator.getNewHostsWithInconsistentTopologiesAndRetry(any(), any(), any(), any()))
+                .thenReturn(Set.of());
```

Comment:
```suggestion
        setupHostsWithInconsistentTopology(Set.of())
```
If you do this, then ignore the suggestion to use varargs (or have two overloaded methods), since Set.of() is clearer than no args.

Reply:
bleh good spot